app/src/main/java/ch/epfl/qedit/model/Question.java

```diff
@@ -12,6 +12,7 @@
 
     private final String text;
 
+    private int index;
```

Comment:
Do we need this field ? I think we removed it in a previous PR, and now we add it again...

Reply:
Actually we don't. I re-added it before seeing the liveData Integer structure you implemented and forgot to remove it, thanks !